src/web/client/common/constants.ts

```diff
@@ -0,0 +1,12 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+export const PORTAL_LANGUAGES = 'adx_portallanguages';
```

Comment:
If we are going to have constants other than Dataverse constants then we should have an Enum to group the constants properly.

Reply:
will take up in next iteration